packages/dd-trace/src/appsec/reporter.js

```diff
@@ -48,6 +46,14 @@ const IDENTIFICATION_HEADERS_MAP = mapHeaderAndTags([
   'x-sigsci-requestid',
   'x-sigsci-tags',
   'akamai-user-risk'
+]
+
+// these request headers are always collected - it breaks the expected spec orders
+const MANDATORY_REQUEST_HEADERS_MAP = mapHeaderAndTags([
+  'content-type',
+  'user-agent',
+  'accept',
+  ...identificationHeaders
```

Comment:
do we really need two objects here ? why not just inline it directly ?

Reply:
I maintained it because I liked the idea of `identificationHeaders` to make it easier to understand, and `content-type`, `user-agent` and `accept` are not `identificationHeaders`.